src/ORM/ValidationResult.php

```diff
@@ -249,6 +250,7 @@ public function serialize()
      */
     public function unserialize($serialized)
     {
+        Deprecation::notice('4.12.0', 'Will be removed in 5.0');
```

Comment:
Add the version to the phpdoc, and replace the message with something standardised

Reply:
Updated